libfwupd/fwupd-enums.c

```diff
@@ -312,6 +314,8 @@ fwupd_device_flag_from_string(const gchar *device_flag)
 		return FWUPD_DEVICE_FLAG_UNREACHABLE;
 	if (g_strcmp0(device_flag, "affects-fde") == 0)
 		return FWUPD_DEVICE_FLAG_AFFECTS_FDE;
+	if (g_strcmp0(device_flag, "no-auto-remove") == 0)
+		return FWUPD_DEVICE_FLAG_NO_AUTO_REMOVE;
```

Comment:
if these end up being internal flags, then this should move to `libfwupdplugin/fu-device.c`

Reply:
let me look at these , still open

Reply:
should I move this flags to libfwupdplugin/fu-device.c ?

Reply:
Yes, move it to fu-device.c and the matching one from fwupd-enums.h to fu-device.h

Reply:
done in https://github.com/fwupd/fwupd/pull/4046/commits/8d0d551cc1a62fbd54d246ec7a5ddb513bac95bd